media/js/src/main.js

```diff
@@ -114,7 +114,7 @@ require([
             zIndexOffset: 1040
         });
 
-        $('input.datepicker').datepicker({
+        $('input .datepicker').datepicker({
```

Comment:
Is this space intended?

Reply:
Hmm, this change _fixed_ the datepicker instantiation for me.  Let me verify the behavior.